packages/cli/src/lifecycles/scaffold.js

```diff
@@ -23,49 +23,61 @@ const writePageComponentsFromTemplate = async (compilation) => {
     });
   };
 
-  const loadPageMeta = async (file, result, { metaComponent }) => {
-    return new Promise((resolve, reject) => {
-      try {
-        const { title, meta, route } = file;
-        const metadata = {
-          title,
-          meta,
-          route
-        };
+  const loadPageMeta = (file, result, { metaComponent }) => {
+    const { title, meta, route } = file;
+    const metadata = {
+      title,
+      meta,
+      route
+    };
 
-        result = result.replace(/METAIMPORT/, `import '${metaComponent}'`);
-        result = result.replace(/METADATA/, `const metadata = ${JSON.stringify(metadata)}`);
-        result = result.replace(/METAELEMENT/, '<eve-meta .attributes=\${metadata}></eve-meta>');
+    result = result.replace(/METAIMPORT/, `import '${metaComponent}'`);
+    result = result.replace(/METADATA/, `const metadata = ${JSON.stringify(metadata)}`);
+    result = result.replace(/METAELEMENT/, '<eve-meta .attributes=\${metadata}></eve-meta>');
 
-        resolve(result);
+    return result;
+  };
+
+  const writeComponentToFile = async (target, filename, result) => {
+    return new Promise(async(resolve, reject) => {
+      try {
+        await fs.ensureDir(target, { recursive: true });
+        await fs.writeFile(path.join(target, `${filename}.js`), result);
+        resolve();
       } catch (err) {
         reject(err);
       }
     });
   };
 
+  const determineTarget = (file, context) => {
```

Comment:
what is target in this case?  Do we mean output directory / path?  In NodeJS land, target may be more associated with platform, like browser, nodejs, electron, etc.

I think if what we're doing is more filesystem related, we should stick that more familiar taxonomy / language where possible.

Reply:
Fair point, it's vague.